backoffice/certificate_manager/utils.py

```diff
@@ -89,9 +90,26 @@ def generate_fun_certificate(student, course, teachers, university):
 
         set_certificate_filename(cert, certificate_filename)
     cert.save()
+
+    trigger_tracking_log(cert, course, student)
+
     return cert.status
 
 
+def trigger_tracking_log(cert, course, student):
+    """
+    Log the certificate generation for a certificate, a course and a student in
+    EDX tracking logs (same process as in EDX).
+    """
+    if cert.status in [CertificateStatuses.generating, CertificateStatuses.downloadable]:
```

Comment:
Je fais comme fait edx. J'ai peur que loger les échecs rajoute beaucoup de
bruits aux tl...

Le ven. 20 mai 2016 10:50, Richard Moch <notifications@github.com> a écrit :

> In backoffice/certificate_manager/utils.py
> <https://github.com/openfun/fun-apps/pull/510#discussion_r64008742>:
>
> >      return cert.status
> >
> >
> > +def trigger_tracking_log(cert, course, student):
> > +    """
> > +    Log the certificate generation for a certificate, a course and a student in
> > +    EDX tracking logs (same process as in EDX).
> > +    """
> > +    if cert.status in [CertificateStatuses.generating, CertificateStatuses.downloadable]:
>
> Pourquoi ne pas logger les échecs ? (status 'notpassing')
> Et dans quel cas on va arriver là avec un status 'generating' ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openfun/fun-apps/pull/510/files/b94586a8820aca8b4a661bba3cd11e5b13e519bb#r64008742>
>